app/helpers/application_helper/button/collect_logs.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationHelper::Button::CollectLogs < ApplicationHelper::Button::Basic
+  def calculate_properties
+    super
+    if @record.try(:log_file_depot).try(:requires_support_case?)
+      self[:prompt] = true
```

Comment:
unfortunately not :-(

in some of the next steps I will try to fix this and have all keys unified